packages/braintree-utils/src/index.ts

```diff
@@ -1,37 +1,8 @@
-export {
-    BraintreeModuleCreator,
-    BraintreeInitializationData,
-    BraintreeTokenizePayload,
-    BraintreeError,
-    BraintreeDataCollector,
-    BraintreePaypalCheckoutCreator,
-    BraintreePaypalCheckout,
-    BraintreePaypalSdkCreatorConfig,
-    BankAccountSuccessPayload,
-    BraintreeBankAccount,
-    BraintreeConnect,
-    BraintreeConnectAuthenticationState,
-    BraintreeConnectAddress,
-    BraintreeConnectVaultedInstrument,
-    BraintreeConnectCardComponentOptions,
-    BraintreeConnectCardComponent,
-    BraintreeConnectPhone,
-    BraintreeConnectProfileData,
-    onPaymentStartData,
-    StartPaymentError,
-    LocalPaymentsPayload,
-    LocalPaymentInstance,
-    BraintreeHostWindow,
-} from './braintree';
-
+export * from './braintree';
```

Comment:
are you sure we need to export all from `./braintree`? i mean a lot of redundant types are exported too and never be used in another place

Reply:
This `*` is returning only exported interfaces (`export interface FooBar { }`) from the file. Therefore it should provide only those interfaces that should be useful in other places, otherwise we should ask ourself: why do we export an interface that is not in use anywhere?

So yes, I think we can use `*` to export all interfaces in the file and save some time on missed interface exports for future development, while working with `./braintree` file.